explorer/interpreter/action.h

```diff
@@ -121,6 +121,32 @@ class Action {
   // object.
   auto kind() const -> Kind { return kind_; }
 
+  auto kind_string() -> std::string_view {
```

Comment:
It looks like you have some changes from the other PR mixed in here.

Reply:
Sorry, I missed removing it. Will do in the next commit.